.github/workflows/main.yml

```diff
@@ -50,15 +50,16 @@ jobs:
           key: |
             modules-${{ hashFiles('**/package-lock.json') }}
 
+      - name: Install dependencies
+        if: steps.cache.outputs.cache-node_modules != 'true'
+        run: npm ci --ignore-scripts
+
       - name: Cache Dojo
         id: cache-dojo
         uses: actions/cache@v2
         with:
           path: UI/js
-          key: dojo-${{ hashFiles('UI/{js-src,css}/**') }}
-
-      - name: Install Node modules Dependencies
-        run: npm ci
+          key: dojo-${{ hashFiles('UI/{js-src,css}/**') }}-${{ hashFiles('UI/**/*.html') }}
```

Comment:
You can use multiple patterns with a single hashfiles invocation too: https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#example-with-multiple-patterns

Reply:
Thanks for the heads-up, that was forbibben before